modules/appeals_api/app/workers/appeals_api/daily_stuck_records_report.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sidekiq/monitored_worker'
+
+class AppealsApi::DailyStuckRecordsReport
+  include Sidekiq::Worker
+  include Sidekiq::MonitoredWorker
+
+  # Only retry for ~8 hours since the job runs daily
+  sidekiq_options retry: 11, unique_for: 8.hours
+
+  def perform
+    params = find_stuck_record_data(AppealsApi::HigherLevelReview) + \
+             find_stuck_record_data(AppealsApi::NoticeOfDisagreement) + \
+             find_stuck_record_data(AppealsApi::SupplementalClaim)
+
+    AppealsApi::Slack::Messager.new(params, notification_type: :stuck_record).notify!
+  end
+
+  def enabled?
+    Settings.vsp_environment == 'production' &&
+      Settings.modules_appeals_api.reports.daily_stuck_records.enabled
+  end
+
+  def retry_limits_for_notification
+    [11]
+  end
+
+  def notify(retry_params)
+    AppealsApi::Slack::Messager.new(retry_params, notification_type: :error_retry).notify!
+  end
+
+  private
+
+  def find_stuck_record_data(klass)
+    attrs = %i[id status created_at]
+    klass.stuck&.find_each.map do |record|
```

Comment:
find_each accepts a block that yields the record - the extra `.map` is redundant and possibly negates some of find_each's benefit. Just remove `.map` but keep the block the same and you should be good!